addons/pos_sale/static/src/app/order_management_screen/sale_order_management_screen/sale_order_management_screen.js

```diff
@@ -444,15 +444,6 @@ export class SaleOrderManagementScreen extends ControlButtonsMixin(Component) {
         const sale_lines = await this._getSOLines(sale_order.order_line);
         sale_order.order_line = sale_lines;
 
-        if (sale_order.picking_ids[0]) {
-            const [picking] = await this.orm.read(
-                "stock.picking",
-                [sale_order.picking_ids[0]],
-                ["scheduled_date"]
-            );
-            sale_order.shipping_date = picking.scheduled_date;
-        }
-
```

Comment:
This seems to me a feature. Maybe we need to check why it is introduced.

Reply:
The only place it's used is for the ship later button. That's why it was selecting the date by default. So we just need to remove this so no ship later date is selected by default

Reply:
It's introduced here: https://github.com/odoo/odoo/commit/fe9e230b500af316e6f80af453a4073d9dff24b7 can you ask ADGU to be sure.